docs/wiki/basic-tutorials/embeds.md

```diff
@@ -39,6 +39,34 @@ EmbedBuilder embed = new EmbedBuilder()
 // Send the embed
 channel.sendMessage(embed);
 ```
+All this is a bit confusing :thinking: so let's break it down!
+
+## :bone: Basics
+To start using the embed builder, you need to instantiate it like any other class and then you can start acessing it's methods. We'll go through the basic ones here:
+
+* `EmbedBuilder.setTitle("")`- Most of the time you will want a title for your embed, won't you :question: This sets the title of the embed, the top-most words in the embed.
+
+* `EmbedBuilder.setDescription("")`- This sets the description, this is like the subtitle of your embed if you will. It is displayed right after the title.
+
+* `EmbedBuilder.addFeild("", "", false)`- So if you want more subtitles than your description, This is the way to do that! thats cool and all, but what are the 3 parameters? The first one is the subtitle of the feild and the second one is the content of that subtitle, and the third one is a boolean that will change the position of the subtitle, suppose there are 2 of these "feilds" and you want them to go side by side then just set both (or any one) to have the inline value (the last parameter) to true and it will display it on the same line! Alternatively you can also use the dedicated method, `EmbedBuilder.addInlineFeild("");` to do the work of the last parameter in the `EmbedBuilder.addFeild("")`
```

Comment:
Maybe mention here that one can achieve empty fields with invisible characters like "\u200B"